### PR TITLE
Update TiDB Serverless limitations

### PR DESCRIPTION
**Pull request by sunxiaoguang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by sunxiaoguang** [comment]
Link checked failed because it depends another document that's also working in progress.

See https://github.com/pingcap/docs/pull/13967

File: tidb-cloud/serverless-tier-limitations.md [hunk 1]
```diff
@@ -7,49 +7,53 @@ summary: Learn about the limitations of TiDB Serverless.
...
-- [Third-party Monitoring integrations](/tidb-cloud/third-party-monitoring-integrations.md) are currently not available for TiDB Serverless.
-- [Cluster Events](/tidb-cloud/tidb-cloud-events.md) are currently not available for TiDB Serverless.
-- [Built-in Alerting](/tidb-cloud/monitor-built-in-alerting.md) is currently not available for TiDB Serverless.
+- [Third-party Monitoring integrations](/tidb-cloud/third-party-monitoring-integrations.md) are currently unavailable.
+- [Cluster Events](/tidb-cloud/tidb-cloud-events.md) are currently unavailable.
+- [Built-in Alerting](/tidb-cloud/monitor-built-in-alerting.md) is currently unavailable.
+- [Key Visualizer](/tidb-cloud/tune-performance.md#key-visualizer) is currently unavailable.
 
-### Diagnosis
+### Self service upgrades
```
Comment (shizn) [inline tidb-cloud/serverless-tier-limitations.md:42]: Self-service

File: tidb-cloud/serverless-tier-limitations.md [hunk 2]
```diff
@@ -7,49 +7,53 @@ summary: Learn about the limitations of TiDB Serverless.
...
+- [Changefeed](/tidb-cloud/changefeed-overview.md) is currently unavailable.
+- [Data Migration](/tidb-cloud/migrate-from-mysql-using-data-migration.md) is currently unavailable.
 
-### Maintenance window
+### Others
 
-- [Maintenance window](/tidb-cloud/configure-maintenance-window.md) is unavailable for TiDB Serverless.
+- [Time to live (TTL)](/time-to-live.md) is currently unavailable.
+- Transaction can not last longer than 30m
+- Total size of a single transaction is set to no more than XX MB
```
Comment (shizn) [inline tidb-cloud/serverless-tier-limitations.md:55]: Please update xx with the actual number.

Comment (sunxiaoguang) [reply tidb-cloud/serverless-tier-limitations.md]: Confirmed that the limitation is gone.

File: tidb-cloud/serverless-tier-limitations.md (hunk 1 above)
Comment (sunxiaoguang) [reply tidb-cloud/serverless-tier-limitations.md]: Fixed

File: tidb-cloud/serverless-limitations.md [hunk 3]
```diff
@@ -8,49 +8,51 @@ aliases: ['/tidbcloud/serverless-tier-limitations']
...
 
-This document describes the limitations of TiDB Serverless.
+TiDB Serverless works with almost all workloads that TiDB supports, but there are feature differences between TiDB Self-Hosted or TiDB Dedicated clusters and TiDB Serverless clusters. This document describes the limitations of TiDB Serverless. 
 
 We are constantly filling in the feature gaps between TiDB Serverless and TiDB Dedicated. If you require these features or capabilities in the gap, use [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) or [contact us](https://www.pingcap.com/contact-us/?from=en) for a feature request.
 
 ## Limitations
 
-### SQL
+### Audit Logs
```
Comment (ran-huang) [inline tidb-cloud/serverless-limitations.md:17]:
```suggestion
### Audit logs
```

File: tidb-cloud/serverless-limitations.md [hunk 4]
```diff
@@ -8,49 +8,51 @@ aliases: ['/tidbcloud/serverless-tier-limitations']
...
 
-- Tables `CLUSTER_SLOW_QUERY`, `SLOW_QUERY`, `CLUSTER_STATEMENTS_SUMMARY`, `CLUSTER_STATEMENTS_SUMMARY_HISTORY`, `STATEMENTS_SUMMARY`, `STATEMENTS_SUMMARY_HISTORY` are not available for TiDB Serverless clusters.
+- Only [Public Endpoint](/tidb-cloud/connect-via-standard-connection-serverless.md) and [Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections-serverless.md) can be used. You cannot use [VPC Peering](/tidb-cloud/set-up-vpc-peering-connections.md) to connect to TiDB Serverless clusters. 
+- No "IP Access List" support.
 
-### Transaction
+### Encryption
 
-- The total size of a single transaction is set to no more than 10 MB on TiDB Serverless during the beta phase.
+- Data persisted in your TiDB Serverless cluster undergoes encryption utilizing the encryption tool provided by the cloud provider responsible for managing your cluster. However, TiDB Serverless does not provide additional optional measures for protecting data at-rest on disks beyond infrastructure-level encryption.
```
Comment (ran-huang) [inline tidb-cloud/serverless-limitations.md:29]:
```suggestion
- Data persisted in your TiDB Serverless cluster is encrypted using the encryption tool provided by the cloud provider that manages your cluster. However, TiDB Serverless does not provide any additional optional measures for protecting data at-rest on disks beyond infrastructure-level encryption.
```

File: tidb-cloud/serverless-limitations.md [hunk 5]
```diff
@@ -8,49 +8,51 @@ aliases: ['/tidbcloud/serverless-tier-limitations']
...
 
-### Connection
+### Maintenance window
 
-- Only [Standard Connection](/tidb-cloud/connect-via-standard-connection-serverless.md) and [Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections-serverless.md) can be used. You cannot use [VPC Peering](/tidb-cloud/set-up-vpc-peering-connections.md) to connect to TiDB Serverless clusters.
-- No "IP Access List" support.
+- [Maintenance window](/tidb-cloud/configure-maintenance-window.md) is currently unavailable.
 
-### Monitoring
+### Monitoring and Diagnosis
```
Comment (ran-huang) [inline tidb-cloud/serverless-limitations.md:36]:
```suggestion
### Monitoring and diagnosis
```

File: tidb-cloud/serverless-limitations.md [hunk 6]
```diff
@@ -8,49 +8,51 @@ aliases: ['/tidbcloud/serverless-tier-limitations']
...
-- [Built-in Alerting](/tidb-cloud/monitor-built-in-alerting.md) is currently not available for TiDB Serverless.
+- [Third-party Monitoring integrations](/tidb-cloud/third-party-monitoring-integrations.md) are currently unavailable.
+- [Built-in Alerting](/tidb-cloud/monitor-built-in-alerting.md) is currently unavailable.
+- [Key Visualizer](/tidb-cloud/tune-performance.md#key-visualizer) is currently unavailable.
 
-### Diagnosis
+### Self-service upgrades
 
-- [Key Visualizer](/tidb-cloud/tune-performance.md#key-visualizer) is unavailable for TiDB Serverless.
+- TiDB Serverless is a fully managed deployment of TiDB. Major and minor version upgrades of TiDB are handled by TiDB Cloud, therefore can not be initiated by users.
```
Comment (ran-huang) [inline tidb-cloud/serverless-limitations.md:44]:
```suggestion
- TiDB Serverless is a fully managed deployment of TiDB. Major and minor version upgrades of TiDB are handled by TiDB Cloud and therefore cannot be initiated by users.
```

File: tidb-cloud/serverless-limitations.md [hunk 7]
```diff
@@ -8,49 +8,51 @@ aliases: ['/tidbcloud/serverless-tier-limitations']
...
 
 * [Changefeed](/tidb-cloud/changefeed-overview.md) is not supported for TiDB Serverless currently.
 * [Data Migration](/tidb-cloud/migrate-from-mysql-using-data-migration.md) is not supported for TiDB Serverless currently.
 
-### Maintenance window
+### Others
 
-- [Maintenance window](/tidb-cloud/configure-maintenance-window.md) is unavailable for TiDB Serverless.
+- [Time to live (TTL)](/time-to-live.md) is currently unavailable.
+- Transaction can not last longer than 30m
```
Comment (ran-huang) [inline tidb-cloud/serverless-limitations.md:54]:
```suggestion
- Transaction can not last longer than 30 minutes.
```

File: tidb-cloud/serverless-limitations.md [hunk 8]
```diff
@@ -8,49 +8,51 @@ aliases: ['/tidbcloud/serverless-tier-limitations']
...
+- [Limited SQL features](/tidb-cloud/limited-sql-features.md) for more details about SQL limitations.
```
Comment (ran-huang) [inline tidb-cloud/serverless-limitations.md:55]:
```suggestion
- For more details about SQL limitations, refer to [Limited SQL Features](/tidb-cloud/limited-sql-features.md).
```

File: tidb-cloud/serverless-limitations.md [hunk 9]
```diff
@@ -8,49 +8,51 @@ aliases: ['/tidbcloud/serverless-tier-limitations']
...
 ## Limitations
 
-### SQL
+### Audit Logs
 
-- [Time to live (TTL)](/time-to-live.md) is not available for TiDB Serverless clusters currently.
-- The [`FLASHBACK CLUSTER TO TIMESTAMP`](/sql-statements/sql-statement-flashback-to-timestamp.md) syntax is not applicable to [TiDB Serverless](/tidb-cloud/select-cluster-tier.md#tidb-serverless-beta) clusters.
-- The [`SLEEP()` function](/functions-and-operators/miscellaneous-functions.md) only supports a maximum sleep time of 300 seconds.
+- Database audit logging is currently unavailable.
+- Console audit logging is currently unavailable.
```
Comment (ran-huang) [inline tidb-cloud/serverless-limitations.md:20]: Please add a note on https://docs.pingcap.com/tidbcloud/tidb-cloud-console-auditing saying that it is unavailable for serverless

File: tidb-cloud/serverless-limitations.md [hunk 10]
```diff
@@ -8,49 +8,51 @@ aliases: ['/tidbcloud/serverless-tier-limitations']
...
 
-### Monitoring
+### Monitoring and Diagnosis
 
-- [Third-party Monitoring integrations](/tidb-cloud/third-party-monitoring-integrations.md) are currently not available for TiDB Serverless.
-- [Cluster Events](/tidb-cloud/tidb-cloud-events.md) are currently not available for TiDB Serverless.
-- [Built-in Alerting](/tidb-cloud/monitor-built-in-alerting.md) is currently not available for TiDB Serverless.
+- [Third-party Monitoring integrations](/tidb-cloud/third-party-monitoring-integrations.md) are currently unavailable.
+- [Built-in Alerting](/tidb-cloud/monitor-built-in-alerting.md) is currently unavailable.
+- [Key Visualizer](/tidb-cloud/tune-performance.md#key-visualizer) is currently unavailable.
```
Comment (ran-huang) [inline tidb-cloud/serverless-limitations.md:40]: Is [Index Insight](https://docs.pingcap.com/tidbcloud/tune-performance#index-insight-beta) available for Serverless? Its doc says **only available for Dedicated**.

File: tidb-cloud/serverless-limitations.md (hunk 9 above)
Comment (sunxiaoguang) [reply tidb-cloud/serverless-limitations.md]: Thanks for pointing out. Looks like this console auditing is available for serverless. It's fixed now.

File: tidb-cloud/serverless-limitations.md (hunk 10 above)
Comment (sunxiaoguang) [reply tidb-cloud/serverless-limitations.md:39]: Added
In addition,  [Cluster Events](https://docs.pingcap.com/tidbcloud/tidb-cloud-events) is unavailable as well.

**Comment by sunxiaoguang** [comment]
Links check failed as the one of the modified document has link requires login. Since it's not related to changes, it can be ignored.

**Review by ran-huang — APPROVED** [review]

File: tidb-cloud/limited-sql-features.md [hunk 11]
```diff
@@ -0,0 +1,246 @@
+---
+title: Limited SQL Features on TiDB Cloud
+summary: Learn about the limited SQL features on TiDB Cloud.
+---
+
+# Limited SQL features on TiDB Cloud
+
+TiDB Cloud works with almost all workloads that TiDB supports, but there are feature differences between TiDB Self-Hosted and TiDB Dedicated/Serverless. This document describes the limitations of SQL features on TiDB Cloud. We are constantly filling in the feature gaps between TiDB Self-Hosted and TiDB Dedicated/Serverless. If you require these features or capabilities in the gap, [contact us](/tidb-cloud/tidb-cloud-support.md) for a feature request.
```
Comment (qiancai) [inline tidb-cloud/limited-sql-features.md:8]:
```suggestion
TiDB Cloud works with almost all workloads that TiDB supports, but there are some feature differences between TiDB Self-Hosted and TiDB Dedicated/Serverless. This document describes the limitations of SQL features on TiDB Cloud. We are constantly filling in the feature gaps between TiDB Self-Hosted and TiDB Dedicated/Serverless. If you require these features or capabilities in the gap, [contact us](/tidb-cloud/tidb-cloud-support.md) for a feature request.
```

File: tidb-cloud/limited-sql-features.md [hunk 12]
```diff
@@ -0,0 +1,246 @@
...
+| `validate_password.number_count` | No limitation | Can not be smaller than 1 [^10] |
+| `validate_password.policy` | No limitation | Can only be MEDIUM or STRONG [^10] |
+| `validate_password.special_char_count` | No limitation | Can not be smaller than 1 [^10] |
+| `wait_timeout` | No limitation | Read-only [^11] |
+
+[^1]: Configuring data placement is not supported on TiDB Serverless.
+
+[^2]: Configuring resource groups is not supported on TiDB Serverless.
+
+[^3]: To perform [Back up and Restore](/tidb-cloud/backup-and-restore-serverless.md) operations on TiDB Serverless, please use the web console instead.
```
Comment (qiancai) [inline tidb-cloud/limited-sql-features.md:233]:
```suggestion
[^3]: To perform [Back up and Restore](/tidb-cloud/backup-and-restore-serverless.md) operations on TiDB Serverless, you can use the TiDB Cloud console instead. 
```

**Comment by sunxiaoguang** [comment]
hold on for a sec. There is a limited SQL need to be verified before merge.

File: tidb-cloud/limited-sql-features.md [hunk 13]
```diff
@@ -0,0 +1,246 @@
...
+| `tidb_ttl_scan_batch_size` | No limitation | Read-only [^9] |
+| `tidb_ttl_scan_worker_count` | No limitation | Read-only [^9] |
+| `tidb_txn_mode` | No limitation | Read-only [^11] |
+| `tidb_wait_split_region_finish` | No limitation | Read-only [^11] |
+| `tidb_wait_split_region_timeout` | No limitation | Read-only [^11] |
+| `txn_scope` | No limitation | Read-only [^11] |
+| `validate_password.enable` | No limitation | Always enabled [^10] |
+| `validate_password.length` | No limitation | Can not be smaller than 8 [^10] |
+| `validate_password.mixed_case_count` | No limitation | Can not be smaller than 1 [^10] |
+| `validate_password.number_count` | No limitation | Can not be smaller than 1 [^10] |
```
Comment (qiancai) [inline tidb-cloud/limited-sql-features.md:224]:
```suggestion
| `validate_password.length` | No limitation | At least `8` [^10] |
| `validate_password.mixed_case_count` | No limitation | At least `1` [^10] |
| `validate_password.number_count` | No limitation | At least `1` [^10] |
```

File: tidb-cloud/limited-sql-features.md [hunk 14]
```diff
@@ -0,0 +1,246 @@
...
+| `validate_password.policy` | No limitation | Can only be MEDIUM or STRONG [^10] |
+| `validate_password.special_char_count` | No limitation | Can not be smaller than 1 [^10] |
```
Comment (qiancai) [inline tidb-cloud/limited-sql-features.md:226]:
```suggestion
| `validate_password.special_char_count` | No limitation | At least `1` [^10] |
```

File: tidb-cloud/limited-sql-features.md [hunk 15]
```diff
@@ -0,0 +1,246 @@
...
+| `tidb_ttl_scan_worker_count` | No limitation | Read-only [^9] |
+| `tidb_txn_mode` | No limitation | Read-only [^11] |
+| `tidb_wait_split_region_finish` | No limitation | Read-only [^11] |
+| `tidb_wait_split_region_timeout` | No limitation | Read-only [^11] |
+| `txn_scope` | No limitation | Read-only [^11] |
+| `validate_password.enable` | No limitation | Always enabled [^10] |
+| `validate_password.length` | No limitation | Can not be smaller than 8 [^10] |
+| `validate_password.mixed_case_count` | No limitation | Can not be smaller than 1 [^10] |
+| `validate_password.number_count` | No limitation | Can not be smaller than 1 [^10] |
+| `validate_password.policy` | No limitation | Can only be MEDIUM or STRONG [^10] |
```
Comment (qiancai) [inline tidb-cloud/limited-sql-features.md:225]:
```suggestion
| `validate_password.policy` | No limitation | Can only be `MEDIUM` or `STRONG` [^10] |
```

File: tidb-cloud/limited-sql-features.md [hunk 16]
```diff
@@ -0,0 +1,246 @@
...
+
+[^1]: Configuring data placement is not supported on TiDB Serverless.
+
+[^2]: Configuring resource groups is not supported on TiDB Serverless.
+
+[^3]: To perform [Back up and Restore](/tidb-cloud/backup-and-restore-serverless.md) operations on TiDB Serverless, please use the web console instead. 
+
+[^4]: The feature is unavailable in [Security Enhanced Mode (SEM)](/system-variables.md#tidb_enable_enhanced_security).
+
+[^5]: To track [Slow Query](/tidb-cloud/tune-performance.md#slow-query) on TiDB Serverless, please use the web console instead.
```
Comment (qiancai) [inline tidb-cloud/limited-sql-features.md:237]:
```suggestion
[^5]: To track [Slow Query](/tidb-cloud/tune-performance.md#slow-query) on TiDB Serverless, you can use the TiDB Cloud console instead.
```

File: tidb-cloud/limited-sql-features.md [hunk 17]
```diff
@@ -0,0 +1,246 @@
...
+
+[^6]: To perform [Statement Analysis](/tidb-cloud/tune-performance.md#statement-analysis) on TiDB Serverless, please use the web console instead.
```
Comment (qiancai) [inline tidb-cloud/limited-sql-features.md:239]:
```suggestion
[^6]: To perform [Statement Analysis](/tidb-cloud/tune-performance.md#statement-analysis) on TiDB Serverless, you can use the TiDB Cloud console instead.
```

**Comment by sunxiaoguang** [comment]
> Hold on, There is a limited SQL need to be verified before merge.

Done.

File: tidb-cloud/limited-sql-features.md [hunk 18]
```diff
@@ -0,0 +1,246 @@
...
+### Others
+
+| Statement | TiDB Dedicated | TiDB Serverless |
+|:-|:-|:-|
+| `BACKUP` | Supported | Not supported [^3] |
+| `SHOW BACKUPS` | Supported | Not supported [^3] |
+| `RESTORE` | Supported | Not supported [^3] |
+| `SHOW RESTORES` | Supported | Not supported [^3] |
+| `ADMIN RESET TELEMETRY_ID` | Supported | Telemetry is not supported on TiDB Serverless. |
+| `ADMIN SHOW TELEMETRY` | Not supported [^4] | Not supported [^4] |
```
Comment (qiancai) [inline tidb-cloud/limited-sql-features.md:46]: Is `[^4]` here correct for the TiDB Serverless column?   The previous row indicates that Telemetry is not supported on TiDB Serverless.

Comment (sunxiaoguang) [reply tidb-cloud/limited-sql-features.md:46]: Yes, the second SQL is disabled by SEM therefore is not supported on both dedicated and serverless.

`ADMIN RESET TELEMETRY_ID` is not restricted by SEM, therefor works on dedicated and we explicitly disabled it on serverless.

**Review by qiancai — APPROVED** [review]
/lgtm

**Pull request merged**